### PR TITLE
Restructure IA

### LINKED ISSUE #123: IA Restructure
The IA has been restructured and now includes a `Users` section and `Solutions` has been removed. Subsequent pages have been re-distributed or removed.

- [x] create `/users`
- [x] `/discover/customers` becomes `/discover/use-cases` and update any links/urls
- [x] remove `/solutions` and update any links/urls pointing at `/solutions`
- [x] move `/developers/documentation` to `/users/documentation` and update any links/urls
- [x] move `/developers/get-started` to `/users/get-started` and update any links/urls
- [x] move `/solutions/case-studies` to `/discover/case-studies` and update any links/urls
- [x] move `/solutions/training-professional-services` to `/users/training-professional-services` and update any links/urls
- [x] remove `/foundation/ceph-team` and update any links/urls pointing at `/foundation/ceph-team`

[latest IA](https://miro.com/app/board/o9J_kpb4VwI=/?moveToWidget=3074457357974192318&cot=14)

### PR DESCRIPTION
**Pull request by Pete-Robelou**
- Users section
- Removed Solutions section
- Re-ordered existing pages taking into account new and removed sections

Closes #123

### CODE REVIEW

File: src/en-GB/developers/index.html [hunk 1]
```diff
@@ -28,7 +28,7 @@ <h2 class="h2 mb-12 lg:mb-16">Start using Ceph today</h2>
           software-defined storage system. Becoming a Ceph developer means you belong to a global community of passionate, talented and
           inspired developers, with the common goal of making Ceph the best it can be.
         </p>
-        <a class="button" href="/{{ locale }}/developers/documentation/">Get started with Ceph</a>
+        <a class="button" href="/{{ locale }}/users/documentation/">Get started with Ceph</a>
```
Comment (adamduncan) [inline src/en-GB/developers/index.html:31]: Let's keep an eye on this one. I'm not sure the original intent was to do so much cross-linking between sections in the site (i.e. the "Getting started" page used to live here under Developers > Documentation). But let's see once we start to get more content through

File: src/en-GB/users/index.html [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+            <img alt="" class="absolute bottom-0 left-0 m-4 w-12 z-1" src="/assets/svgs/icon-code-white-red.svg" />
+          </div>
+          <div class="p-5">
+            <h3 class="h3">Code</h3>
+            <p class="p">
+              Explore and create novel, innovative functions with the support and expertise of a global community. Whether you are new to
+              coding or a seasoned expert, input is always valuable.
+            </p>
+            <p class="p">Find out where Ceph is headed and where your coding skill could be applied.</p>
+            <a class="a link-cover link-cover--shadow" href="/{{ locale }}/developers/code/">Code Ceph</a>
```
Comment (adamduncan) [inline src/en-GB/users/index.html:81]: Similarly here, we'll want to confirm which sections/pages we want to highlight here on the Users hub, either from the Users section itself or other site sections

**Review by adamduncan — APPROVED** [review]
Thanks @Pete-Robelou. All looks good to merge 👍 

Just a couple comments re: cross-linking now the sections have been restructured. I expect we'll revisit these anyway as content is gathered for the Users/Developers sections. cc @Thingee

**Comment by adamduncan** [comment]
@Pete-Robelou Also, just ran a quick link checker across the site. All look good.

Just one it threw up [here](https://github.com/ceph/ceph.io/blob/develop/src/en-GB/community/index.html#L108): we have a `tect-talks` link, which should be `tech-talks`. Thanks

**Review by liewegas — APPROVED** [review]
Yay!

File: src/en-GB/developers/index.html (hunk 1 above)
Comment (Pete-Robelou) [reply src/en-GB/developers/index.html:31]: Agreed. As more content flows in we can adjust the cross-linking as necessary.

File: src/en-GB/users/index.html (hunk 2 above)
Comment (Pete-Robelou) [reply src/en-GB/users/index.html:81]: Agreed. As more content flows in we can adjust the cross-linking as necessary.

**Comment by Pete-Robelou** [comment]
> @Pete-Robelou Also, just ran a quick link checker across the site. All look good.
> 
> Just one it threw up [here](https://github.com/ceph/ceph.io/blob/develop/src/en-GB/community/index.html#L108): we have a `tect-talks` link, which should be `tech-talks`. Thanks

Fixed 🙂

**Pull request merged**